app/controllers/application_controller.rb

```diff
@@ -1,7 +1,20 @@
 class ApplicationController < ActionController::Base
-  before_action :authenticate_user!
+  include Pundit
+  after_action :verify_authorized, unless: :devise_controller?
+  after_action :verify_policy_scoped, only: :index, unless: :devise_controller?
 
+  before_action :authenticate_user!
   before_action :configure_permitted_parameters, if: :devise_controller?
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+  #rescue_from Pundit::AuthorizationNotPerformedError, with: :user_not_authorized
```

Comment:
You can get rid of this code you commented out!

Reply:
Will do, thanks Reid! :)